lib/cmd/fh3/appdata/export/status.js

```diff
@@ -0,0 +1,127 @@
+"use strict";
+
+var fs = require('fs')
+  , _ = require('underscore')
+  , fhreq = require('../../../../utils/request.js')
+  , async = require('async')
+  , ProgressBar = require('progress')
+  , common = require('../../../../common');
+
+var POLL_INTERVAL = 2000;
+var MIN_INTERVAL = 100;
+var EXIT_KEYS = ['q', 'Q'];
+
+/**
+ * Read progress information by calling the `read` endpoint to get the
+ * job definition. Progress information is included there:
+ * - step: current export job step
+ * - totalSteps: total export steps
+ */
+function readProgress(params, cb) {
+  var url = 'api/v2/appdata/' + params.envId + '/' + params.appId + '/export/data/' + params.jobId;
+  common.doGetApiCall(fhreq.getFeedHenryUrl(), url, 'Error reading progress', cb);
+}
+
+/**
+ * Update the progressbar if possible. Don't use `bar#tick` as that will not
+ * set the current progress but add the new valud to the current progress.
+ */
+function printout(data, bar) {
+  if (bar && data.totalSteps && data.step >= 0) {
+    bar.update(data.step / data.totalSteps);
+  } else {
+    /* eslint disable-next-line no-console */
+    console.log('No progress information in response');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)